cypress/support/fragments/orders/orderLines.js

```diff
@@ -371,9 +389,9 @@ export default {
   fillInPOLineInfoWithFund: (fund) => {
     cy.do([
       orderLineTitleField.fillIn(orderLineTitle),
-      orderFormatSelect.choose('Physical resource'),
+      orderFormatSelect.choose(ORDER_FORMAT_NAMES.PHYSICAL_RESOURCE),
       acquisitionMethodButton.click(),
-      SelectionOption('Depository').click(),
+      SelectionOption(ACQUISITION_METHOD_NAMES.DEPOSITORY).click(),
       physicalUnitPriceTextField.fillIn(physicalUnitPrice),
       quantityPhysicalTextField.fillIn('2'),
       materialTypeSelect.choose('book'),
```

Comment:
Please use here the variable from the constants file from the support directory

Reply:
Done